core/src/main/java/org/infinispan/factories/SecurityActions.java

```diff
@@ -34,19 +32,10 @@ private static Field findFieldRecursively(Class<?> c, String fieldName) {
    static void setValue(Object instance, String fieldName, Object value) {
```

Comment:
nitpick, this method fits better in `org.infinispan.commons.util.ReflectionUtil` since  `setField` lives there